default/python/AI/FreeOrionAI.py

```diff
@@ -83,29 +68,31 @@ def startNewGame(aggression=fo.aggression.aggressive):  # pylint: disable=invali
         return
 
     turn_timer.start("Server Processing")
-    print "New game started, AI Aggression level %d (%s)" % (aggression, UserString(_aggression_names[aggression]))
 
     # initialize AIstate
     global foAIstate
     print "Initializing foAIstate..."
-    foAIstate = AIstate.AIstate(aggression)
+    foAIstate = AIstate.AIstate(_aggression)
+    aggression = foAIstate.character.get_trait(Aggression)
```

Comment:
Using all 3 ```_aggression, aggression, Aggression``` is kind of unclear. 

Maybe use ```aggression``` as parameter and ```aggression_trait``` for the second variable.